cachi2/core/models/output.py

```diff
@@ -32,6 +43,13 @@ def key(self) -> str:
 
         return f"{self.name}:{self.version}"
 
+    @pydantic.validator("properties", always=True)
+    def _add_found_by_property(cls, properties: list[Property]) -> list[Property]:
+        if FOUND_BY_CACHI2_PROPERTY not in properties:
+            properties.append(FOUND_BY_CACHI2_PROPERTY)
+
+        return properties
```

Comment:
IMO we can append the property unconditionally. I don't see us accidentally adding the `FOUND_BY_CACHI2_PROPERTY` from package manager code ever :smile:

Reply:
Unless that duplicates them when merging the SBOMs of different package managers https://github.com/containerbuildsystem/cachi2/blob/03ee6507b7e06bcd1393ff826982e6d95a89bbd7/cachi2/core/resolver.py#L35

Reply:
It seems that the validation is only triggered a the component is instantiated. Adding it to another model, even in the constructor, does not trigger a validation (at least, it didn't in the test I did).

I would probably just leave the condition there, so we could avoid a corner case.